mains/biryani.md

```diff
@@ -0,0 +1,8 @@
+Half Boiled rice
+Chicken pieces
+Tomatos
+Potatoes
+Spices
+Oil
+Saffron
+This is a mistake!
```

Comment:
Perhaps this line should be commented out

```suggestion
# This is a mistake!
```

Reply:
Thanks.